### PR TITLE
New Uniforms

### PR DESCRIPTION
**Pull request by TyroneMF**
**When merged this pull request will:**
- Add 5 new plaid uniforms
- Replace some old textures

### CODE REVIEW

**Comment by Kresky** [comment]
To stay consistent with the other stuff, I believe the classnames shouldn't have `_CO` at the end of them.
Also seems plaid shirts have a C in front (`BPS` -> `CBPS`), guessing short for "checkered".

In the stringtable, you only describe the color of the shirt, not the pants or boots. Might also want to group them in next to the other plaid stuff, instead of at the top?

Also, what's the reason for replacing some of the old textures? Checked some of them out, and it just seems the new ones you added are more washed out to me? (Haven't seen ingame comparison, but comparing texture files, I prefer the old ones)

**Comment by jonpas** [comment]
Yes on the naming and position.

All replaced textures have been selected by members on Slack when Mick put it up for voting.

**Comment by TyroneMF** [comment]
Everything should be up to standard now, minus the German/Japanese translations.

File: addons/units/CfgWeapons.hpp [hunk 1]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
             uniformClass = QCLASS(Unit_Combat_LS_WS_WP_WB);
         };
     };
+
     // Plaid
+    class CLASS(Uniform_Combat_LS_CBPS_BP_BB): CLASS(Uniform_Base) { // BPS = Buffalo Plaid Shirt
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_LS_CBPS_BP_BB): CLASS(Uniform_Base) { // CBP = Buffalo Plaid
```

File: addons/units/CfgWeapons.hpp [hunk 2]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CBPS_BP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cbps_bp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]: Also, uniform pictures have no boots, so their names are without boots:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_cbps_bp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 3]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CBPS_BP_BB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CDES_TP_TB): CLASS(Uniform_Base) { // DES = Dark Earth Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_LS_CDES_TP_TB): CLASS(Uniform_Base) { // CDE = Plaid Dark Earth
```

File: addons/units/CfgWeapons.hpp [hunk 4]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CDES_TP_TB);
+        picture = QPATHOF(UI\uniform_combat_ls_cdes_tp_tb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_cdes_tp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 5]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CDES_TP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CDFS_GP_TB): CLASS(Uniform_Base) { // DFS = Dark Foliage Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_LS_CDFS_GP_TB): CLASS(Uniform_Base) { // CDF = Plaid Dark Foliage
```

File: addons/units/CfgWeapons.hpp [hunk 6]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CDFS_GP_TB);
+        picture = QPATHOF(UI\uniform_combat_ls_cdfs_gp_tb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_cdfs_gp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 7]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CDFS_GP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_LS_CLES_TP_BB): CLASS(Uniform_Base) { // LES = Light Earth Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_LS_CLES_TP_BB): CLASS(Uniform_Base) { // CLE = Plaid Light Earth
```

File: addons/units/CfgWeapons.hpp [hunk 8]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CLES_TP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cles_tp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_cles_tp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 9]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CLES_TP_BB);
+        };
+    };
+    class CLASS (Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // LFS = Light Foliage Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS (Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // CLF = Light Foliage
```

File: addons/units/CfgWeapons.hpp [hunk 10]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CLFS_GP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_clfs_gp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_ls_clfs_gp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 11]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
             uniformClass = QCLASS(Unit_Combat_RS_TS_TP_TB);
         };
     };
+
     // Plaid
+    class CLASS(Uniform_Combat_RS_CBPS_BP_BB): CLASS(Uniform_Base) { // BPS = Buffalo Plaid Shirt
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_RS_CBPS_BP_BB): CLASS(Uniform_Base) { // CBP = Buffalo Plaid
```

File: addons/units/CfgWeapons.hpp [hunk 12]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CBPS_BP_BB);
+        picture = QPATHOF(UI\uniform_combat_rs_cbps_bp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_rs_cbps_bp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 13]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_RS_CBPS_BP_BB);
+        };
+    };
+    class CLASS(Uniform_Combat_RS_CDES_TP_TB): CLASS(Uniform_Base) { // DES = Dark Earth Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_RS_CDES_TP_TB): CLASS(Uniform_Base) { // CDE = Plaid Dark Earth
```

File: addons/units/CfgWeapons.hpp [hunk 14]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CDES_TP_TB);
+        picture = QPATHOF(UI\uniform_combat_rs_cdes_tp_tb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_rs_cdes_tp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 15]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_RS_CDES_TP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_RS_CDFS_GP_TB): CLASS(Uniform_Base) { // DFS = Dark Foliage Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_RS_CDFS_GP_TB): CLASS(Uniform_Base) { // CDF = Plaid Dark Foliage
```

File: addons/units/CfgWeapons.hpp [hunk 16]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CDFS_GP_TB);
+        picture = QPATHOF(UI\uniform_combat_rs_cdfs_gp_tb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_rs_cdfs_gp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 17]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_RS_CDFS_GP_TB);
+        };
+    };
+    class CLASS(Uniform_Combat_RS_CLES_TP_BB): CLASS(Uniform_Base) { // LES = Light Earth Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_RS_CLES_TP_BB): CLASS(Uniform_Base) { // CLE = Plaid Light Earth
```

File: addons/units/CfgWeapons.hpp [hunk 18]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CLES_TP_BB);
+        picture = QPATHOF(UI\uniform_combat_rs_cles_tp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_rs_cles_tp_ca.paa);
```

File: addons/units/CfgWeapons.hpp [hunk 19]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_RS_CLES_TP_BB);
+        };
+    };
+    class CLASS (Uniform_Combat_RS_CLFS_GP_BB): CLASS(Uniform_Base) { // LFS = Light Foliage Plaid
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS (Uniform_Combat_RS_CLFS_GP_BB): CLASS(Uniform_Base) { // CLF = Plaid Light Foliage
```

File: addons/units/CfgWeapons.hpp [hunk 20]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CLFS_GP_BB);
+        picture = QPATHOF(UI\uniform_combat_rs_clfs_gp_bb_ca.paa);
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
        picture = QPATHOF(UI\uniform_combat_rs_clfs_gp_ca.paa);
```

File: addons/units/stringtable.xml [hunk 21]
```diff
@@ -85,6 +85,26 @@
             <Japanese>戦闘服 長袖 (黄褐、黄褐、黄褐)</Japanese>
             <French>Uniforme de combat long (Beige, Beige, Beige)</French>
         </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_LS_CBPS_BP_BB">
+            <English>Combat Uniform Long (Buffalo Plaid, Black, Black)</English>
+            <French>Uniforme de combat long (Chemise à carreaux, Noir, Noir)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_LS_CDES_TP_TB">
+            <English>Combat Uniform Long (Dark Earth, Tan, Tan)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Long (Plaid Dark Earth, Tan, Tan)</English>
```

File: addons/units/stringtable.xml [hunk 22]
```diff
@@ -85,6 +85,26 @@
...
+            <French>Uniforme de combat long (Terre sombre, Beige, Beige)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_LS_CDFS_GP_TB">
+            <English>Combat Uniform Long (Dark Foliage, Green, Tan)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Long (Plaid Dark Foliage, Green, Tan)</English>
```

File: addons/units/stringtable.xml [hunk 23]
```diff
@@ -85,6 +85,26 @@
...
+            <French>Uniforme de combat long (Feuillage sombre, Vert, Beige)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_LS_CLES_TP_BB">
+            <English>Combat Uniform Long (Light Earth, Tan, Black)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Long (Plaid Light Earth, Tan, Black)</English>
```

File: addons/units/stringtable.xml [hunk 24]
```diff
@@ -85,6 +85,26 @@
...
+            <French>Uniforme de combat long (Terre claire, Beige, Noir)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_LS_CLFS_GP_BB">
+            <English>Combat Uniform Long (Light Foliage, Green, Black)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Long (Plaid Light Foliage, Green, Black)</English>
```

File: addons/units/stringtable.xml [hunk 25]
```diff
@@ -193,6 +213,26 @@
             <Japanese>戦闘服 袖まくり (黄褐、黄褐、黄褐)</Japanese>
             <French>Uniforme de combat retroussé (Beige, Beige, Beige)</French>
         </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_RS_CBPS_BP_BB">
+            <English>Combat Uniform Rolled (Buffalo Plaid, Black, Black)</English>
+            <French>Uniforme de combat retroussé (Chemise à carreaux, Noir, Noir)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_RS_CDES_TP_TB">
+            <English>Combat Uniform Rolled (Dark Earth, Tan, Tan)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Rolled (Plaid Dark Earth, Tan, Tan)</English>
```

File: addons/units/stringtable.xml [hunk 26]
```diff
@@ -193,6 +213,26 @@
...
+            <French>Uniforme de combat retroussé (Terre sombre, Beige, Beige)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_RS_CDFS_GP_TB">
+            <English>Combat Uniform Rolled (Dark Foliage, Green, Tan)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Rolled (Plaid Dark Foliage, Green, Tan)</English>
```

File: addons/units/stringtable.xml [hunk 27]
```diff
@@ -193,6 +213,26 @@
...
+            <French>Uniforme de combat retroussé (Feuillage sombre, Vert, Beige)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_RS_CLES_TP_BB">
+            <English>Combat Uniform Rolled (Light Earth, Tan, Black)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Rolled (Plaid Light Earth, Tan, Black)</English>
```

File: addons/units/stringtable.xml [hunk 28]
```diff
@@ -193,6 +213,26 @@
...
+            <French>Uniforme de combat retroussé (Terre claire, Beige, Noir)</French>
+        </Key>
+        <Key ID="STR_TACS_Units_Uniform_Combat_RS_CLFS_GP_BB">
+            <English>Combat Uniform Rolled (Light Foliage, Green, Black)</English>
```
Comment (jonpas) [inline addons/units/stringtable.xml]:
```suggestion
            <English>Combat Uniform Rolled (Plaid Light Foliage, Green, Black)</English>
```

File: addons/units/CfgVehicles_GroundItems.hpp [hunk 29]
```diff
@@ -125,6 +125,51 @@ class CLASS(Item_Uniform_Combat_LS_TS_TP_TB): Item_Base_F {
         MACRO_ADDITEM(CLASS(Uniform_Combat_LS_TS_TP_TB),1);
     };
 };
+class CLASS(Item_Uniform_Combat_LS_CBPS_BP_BB) : Item_Base_F {
+    MACRO_ITEM_COMMON
+    author = "GilleeDoo, Tyrone";
+    displayName = CSTRING(Uniform_Combat_LS_CBPS_BP_BB);
+
+    class TransportItems {
+        MACRO_ADDITEM(CLASS(Uniform_Combat_LS_CBPS_BP_BB), 1);
```
Comment (jonpas) [inline addons/units/CfgVehicles_GroundItems.hpp]: No space after comma in macro.

File: addons/units/CfgVehicles_HelperUnits.hpp [hunk 30]
```diff
@@ -163,6 +163,36 @@ class CLASS(Unit_Combat_LS_C_Base): B_Soldier_base_F {
     hiddenSelectionsMaterials[] = {QPATHTOF(data\uniform_combat_plaid.rvmat)};
 };
 
+class CLASS(Unit_Combat_LS_CBPS_BP_BB) : CLASS(Unit_Combat_LS_Base) {
```
Comment (jonpas) [inline addons/units/CfgVehicles_HelperUnits.hpp]: No space before `:`.

File: addons/units/CfgVehicles_GroundItems.hpp [hunk 31]
```diff
@@ -125,6 +125,51 @@ class CLASS(Item_Uniform_Combat_LS_TS_TP_TB): Item_Base_F {
         MACRO_ADDITEM(CLASS(Uniform_Combat_LS_TS_TP_TB),1);
     };
 };
+class CLASS(Item_Uniform_Combat_LS_CBPS_BP_BB) : Item_Base_F {
```
Comment (jonpas) [inline addons/units/CfgVehicles_GroundItems.hpp]: No space before `:`.

File: addons/units/CfgWeapons.hpp [hunk 32]
```diff
@@ -192,7 +192,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_LS_CLES_TP_BB);
+        picture = QPATHOF(UI\uniform_combat_ls_cles_tp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_LS_CLES_TP_BB);
+        };
+    };
+    class CLASS (Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // CLF = Light Foliage
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_LS_CLFS_GP_BB_CO): CLASS(Uniform_Base) { // CLF = Light Foliage
```

File: addons/units/CfgWeapons.hpp [hunk 33]
```diff
@@ -365,7 +416,58 @@ class CfgWeapons {
...
+        scope = 2;
+        author = "GilleeDoo, Tyrone";
+        displayName = CSTRING(Uniform_Combat_RS_CLES_TP_BB);
+        picture = QPATHOF(UI\uniform_combat_rs_cles_tp_ca.paa);
+
+        class ItemInfo: ItemInfo {
+            uniformClass = QCLASS(Unit_Combat_RS_CLES_TP_BB);
+        };
+    };
+    class CLASS (Uniform_Combat_RS_CLFS_GP_BB): CLASS(Uniform_Base) { // CLF = Plaid Light Foliage
```
Comment (jonpas) [inline addons/units/CfgWeapons.hpp]:
```suggestion
    class CLASS(Uniform_Combat_RS_CLFS_GP_BB): CLASS(Uniform_Base) { // CLF = Plaid Light Foliage
```

File: addons/units/CfgWeapons.hpp (hunk 32 above)
Comment (Kresky) [reply addons/units/CfgWeapons.hpp]: I believe `_CO` should also be removed, and add "Plaid" to the comment, so:
```suggestion
    class CLASS(Uniform_Combat_LS_CLFS_GP_BB): CLASS(Uniform_Base) { // CLF = Plaid Light Foliage
```
?

Comment (jonpas) [reply addons/units/CfgWeapons.hpp]: No, no "Plaid" in the comment ,those are all plaid. "Buffalo Plaid" is special because it's a type of Plaid.

Comment (jonpas) [reply addons/units/CfgWeapons.hpp]: Well others have it, so I guess it's fine.

**Pull request merged**